src/common/HTTPRequest.ts

```diff
@@ -40,7 +40,10 @@ export interface ContinueRequestOverrides {
  */
 export interface ResponseForRequest {
   status: number;
-  headers: Record<string, string>;
+  /**
+   * Optional response headers. All values are converted to strings.
+   */
+  headers: Record<string, any>;
```

Comment:
Why did you change this to any?

Reply:
Good point, not sure. I think we stringify them anyway so `unknown` is better here.